### PR TITLE
Use prettier to format code

### PR DESCRIPTION
**Pull request by rnestler**

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "trailingComma": "all"
```
Comment (dbrgn) [inline .prettierrc.json]: Should we add `bracketSpacing: false`? https://prettier.io/docs/en/options.html#bracket-spacing I prefer that style.

**Review by dbrgn — CHANGES_REQUESTED** [review]
I'm not a fan of all style choices made by prettier, but that's the goal of the project: An opinionated formatter with very little configuration options (so we can stop arguing about style).

Nevertheless, I'd prefer the _other_ `bracketSpacing` configuration, it's more rust-y 😄

File: .prettierrc.json (hunk 1 above)
Comment (rnestler) [reply .prettierrc.json]: We can do that, generally I changed the options to match the existing style until the diff wasn't that big anymore :wink:

Comment (rnestler) [reply .prettierrc.json]: Done.

**Review by dbrgn — APPROVED** [review]

**Review by netzhuffle — APPROVED** [review]

**Pull request merged**